plugins/Installation/lang/en.json

```diff
@@ -67,6 +67,12 @@
         "SystemCheckCreateFunctionHelp": "Piwik uses anonymous functions for callbacks.",
         "SystemCheckDatabaseExtensions": "MySQL extensions",
         "SystemCheckDatabaseHelp": "Piwik requires either the mysqli extension or both the PDO and pdo_mysql extensions.",
+        "SystemCheckDatabaseSSL": "Database SSL Connection",
+        "SystemCheckDatabaseSSLCipher": "Used cipher",
+        "SystemCheckDatabaseSSLDisabled": "SSL support in your database server is disabled",
+        "SystemCheckDatabaseSSLNo": "Database server is not compiled with SSL support",
+        "SystemCheckDatabaseSSLNotWorking": "%s is set to %s but SSL connection is not working<br \/>",
+        "SystemCheckDatabaseSSLOn": "Database server is supporting SSL connections, please check your configuration in config/config.ini.php <br /> Consider using REQUIRE SSL for your piwik database user",
```

Comment:
Suggested: `Your database supports SSL connections but SSL connection is not used. Check your database SSL settings in your Piwik config file.`